<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">

    <title></title>
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=PT+Serif">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Dancing+Script">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Niconne">
    <link rel="stylesheet" href="resume.css">
  </head>
  <body>
    <div id="container">
  <h1> Adam Conn </h1>
    <div class="header">619 792 8677</div>
    <div class="header">aconn7395@gmail.com</div>

  <h2> Summary </h2>
    <ul id="summary-list">
      <li> Currently a research assistant in a Computational Neurobiology Laboratory since October 2015</li>
      <li> 2.5 years as a wet lab scientists for start up protein purification company </li>
      <li> Published author with current papers in Current Biology and Nature Biosystems</li>
      <li> 2 years experience as a manager for a Starbucks licensed store</li>
      <li> Prior teaching experience in a program focussed on Biotech Instrumentation</li>
      <li> B.S. Degree in Bioengineering: Bioinformatics with a 3.4 GPA </li>
    </ul>
  <h2> Skills </h2>
  <table>
    <tr>
      <th><u> Languages </u></th>
      <th><u> Libraries/ Frameworks</u> </th>
      <th> <u>Algorithms </u></th>
      <th> <u>Software Design/ Other </u></th>
    </tr>
    <tr>
      <td> python </td>
      <td> Django</td>
      <td> Global/local sequence alignment </td>
      <td> Object Oriented Programming </td>
    </tr>
    <tr>
      <td> javascript </td>
      <td> D3 </td>
      <td> De Bruijn Graph Assembly </td>
      <td> Familiarity with Agile process </td>
    </tr>
    <tr>
      <td> java </td>
      <td> sklearn </td>
      <td> Motif finding </td>
      <td> Familiarity with Design Patterns </td>
    </tr>
    <tr>
      <td> html5/css3 </td>
      <td> Django rest framework </td>
      <td> Many classification and regression algoriths </td>
      <td> Familiarity with NGS technologies </td>
    </tr>
  </table>

  <h2> Education </h2>
  <div class="education-item">
    <img src="img/logo_UCSD.png"> B.S. in Bioengineering: Bioinformatics (3.4 GPA) 2015
  </div>
  <div class="education-item">
    <img src="img/city_logo.jpg"> San Diego City College 2012 A.A. in Physics (3.9 GPA)
  </div>
<h2> Work Experience </h2>

  <div class="work-item">
    <div class="job-head">
      <div id="salk-img"></div>
      <div class="job-title"> Research Assistant: October 2015-Current </div>
    </div>
    <ul id="job-description">
      <li> Collect data using high tech 3D laser scanning to capture the architect and mor-
        phology of a plant down to micrometer precision in the form of 3D coordinate point
        clouds.</li>
      <li> Design and write machine learning algorithms to find patterns within the 3D point
        cloud data.</li>
      <li> Write programs to visually view data using matplotlib.
      <li> Implement statistical models that approximate and predict the intricate branching
          structures plants undergo as a response to a specific stressful environment.
      <li> Built an automated pipeline to analyze RNA-SEQ datasets. Involved querying and
        downloading data from NCBI GEO by FTP, normalizing the data, building a binary
        classifier, and extracting the features (genes) used to build our classifier for future
        research.</li>
      <li> Attend presentations, collaborate with other departments and institutions, present
        work, and write sections of articles to be submitted for publication.
        2009-2013</li>

  </div>
  <div class="work-item">
    <div id="mti-img"></div>
  </div>
• Collect data using high tech 3D laser scanning to capture the architect and mor-
phology of a plant down to micrometer precision in the form of 3D coordinate point
clouds.
• Design and write machine learning algorithms to find patterns within the 3D point
cloud data.
• Write programs to visually view data using matplotlib.
• Implement statistical models that approximate and predict the intricate branching
structures plants undergo as a response to a specific stressful environment.
• Built an automated pipeline to analyze RNA-SEQ datasets. Involved querying and
downloading data from NCBI GEO by FTP, normalizing the data, building a binary
classifier, and extracting the features (genes) used to build our classifier for future
research.
• Attend presentations, collaborate with other departments and institutions, present
work, and write sections of articles to be submitted for publication.
2009-2013
• Directly responsible for purifying native and recombinant proteins.
• Routinely worked with multiple cell lines (bacteria/insect/mammalian) to maintain,
transform/transfect, ferment and harvest cells.
• Created novel cDNA libraries.
• Meticulous and detailed documentations
• Performed a wide variety of assays including PCR, RT-PCR, SDS-PAGE, west-
ern blots, DNA-agarose gel electrophoresis, nickel/ion-exchange/size exclusion chro-
matography, cell toxicology and enzyme kinetic assays.
</div>
  <script src="http://localhost:35729/livereload.js"></script>
  </body>
</html>
